autoload/iced/message.vim

```diff
@@ -65,5 +65,22 @@ function! iced#message#error(k, ...) abort
   call iced#message#error_str(msg)
 endfunction
 
+function! iced#message#short(k, ...) abort
```

Comment:
This function should be `iced#util#shorten` because `iced#message`'s role is mainly related to locale setting.
It would be useful for other long messages such as lint error!
https://github.com/liquidz/vim-iced/blob/0.9.1/doc/vim-iced.txt#L1165-L1168